subsys/usb/usb_device.c

```diff
@@ -332,6 +334,11 @@ static void usb_handle_control_transfer(u8_t ep,
 
 		/* Send smallest of requested and offered length */
 		usb_dev.data_buf_residue = MIN(usb_dev.data_buf_len, length);
+		if (usb_dev.data_buf_len > CONFIG_USB_REQUEST_BUFFER_SIZE) {
```

Comment:
This is part of another PR and should not be there. It's not mentioned in PR message.